aroma/utils.py

```diff
@@ -3,14 +3,86 @@
 import os
 import os.path as op
 import shutil
+from tempfile import mkstemp
 
 import nibabel as nib
 import numpy as np
 from nilearn import image, masking
+from nilearn._utils import load_niimg
+from scipy import ndimage
 
 LGR = logging.getLogger(__name__)
 
 
+def load_masks(in_file, csf=None, brain=None):
```

Comment:
@oesteban WDYT of this separation? I realized that having everything in the workflow wouldn't be great, but now there is a dedicated function for the native-space mask generation.

Reply:
seems reasonable 👍